DS/DSAssignment6/CheckFrequency.java

```diff
@@ -0,0 +1,29 @@
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class CheckFrequency {
+
+	private Map<String, Integer> cache = new HashMap<String, Integer>();
+
+	public int checkFrequency(String input) {
+		int output = 0;
+		Set<Character> setOfChatacter = new HashSet<Character>();
```

Comment:
You should instantiate this after the cache statements, because the set will of use on if the cache entry is not there :)